Rohit/PostgresAssignment/src/services/topper.ts

```diff
@@ -0,0 +1,39 @@
+const topSql = require('../database/dbconnect')
+
+exports.get_topper_by_classId_and_subjectId = async (
+  classId: number,
+  subjectId: number
+) => {
+  try {
+    await topSql.query("set search_path to myschool");
+    return await topSql.query(
+      `select s.studentId,s.name,r.marks from (select * from result where class_Id=${classId} 
+      and subject_Id=${subjectId} order by marks desc) as r,students as s
+      where r.studentid=s.studentId limit 1`
+    );
+  } catch (err) {
+    throw err;
+  }
+};
+
+exports.get_topten_students = async (classId: number) => {
+  try {
```

Comment:
why do we need try/catch in services if you are just throwing same error ?

Reply:
done